tests/test_parser/test_mindsdb/test_skills.py

```diff
@@ -0,0 +1,49 @@
+from mindsdb_sql import parse_sql
+from mindsdb_sql.parser.dialects.mindsdb import *
+from mindsdb_sql.parser.ast import *
+
+
+class TestSkills:
+    def test_create_skill(self):
+        sql = '''
+            create skill my_skill
+            using 
+            type = 'knowledge_base', 
+            source ='my_knowledge_base'
+        '''
+        ast = parse_sql(sql, dialect='mindsdb')
+        expected_ast = CreateSkill(
+            name=Identifier('my_skill'),
+            type='knowledge_base',
+            params={'source': 'my_knowledge_base'}
+        )
+        assert str(ast) == str(expected_ast)
+        assert ast.to_tree() == expected_ast.to_tree()
```

Comment:
let's add another round of parsing (after rendering) here to catch problem with render
```python
ast = parse_sql(str(ast), dialect='mindsdb')
assert str(ast) == str(expected_ast)
```

And optional: example with 'IF NOT EXISTS'